test/lib/build/helpers/ProjectBuildContext.js

```diff
@@ -333,10 +336,7 @@ test.serial("getTaskRunner", async (t) => {
 		}
 	}
 	const ProjectBuildContext = await esmock("../../../../lib/build/helpers/ProjectBuildContext.js", {
-		"../../../../lib/build/TaskRunner.js": DummyTaskRunner,
-		"@ui5/logger": {
-			getLogger: sinon.stub().withArgs("type project").returns("log")
-		}
+		"../../../../lib/build/TaskRunner.js": DummyTaskRunner
```

Comment:
The inclusive language check proposes another name for the "DummyTaskRunner". How about "SampleTaskRunner"

Reply:
Done